treeherder/config/settings.py

```diff
@@ -432,3 +433,6 @@
 
 # Only generate alerts for data newer than this time in seconds in perfherder
 PERFHERDER_ALERTS_MAX_AGE = timedelta(weeks=2)
+
+# Resource count to limit the number of threads opening connections with the DB
+CONN_RESOURCES = 50
```

Comment:
You need to merge `master`.

Reply:
Can you specify the hash of commit? Because if I merge master, it will create unnecessary changes, which are not required